src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java

```diff
@@ -45,9 +52,14 @@
     private String menuSessionId;
     private String title;
 
+    // legacy field to be removed once `dateCreated` is fully populated
```

Comment:
This makes sense, just want to make sure you also consider third party hosters. For us we'd be fine deploying it and waiting 8 days or whatever for all old sessions to get deleted, and then deploy code that uses it, but I think we have to expect third parties to probably deploy it once after it's all over so will need some kind of migration from one column to another probably.

Reply:
Yes, I'll make sure to do that.

Reply:
@dannyroberts We delete old form sessions after... 7 days? 14? 

I think it would be a good thing for Solutions and SaaS to get synced up on to have a schedule of how often people are expected to deploy, which would let us set very reasonable fixed dates on this. If we expected teams to deploy formplayer at least once a month we could just set a flag for 45 days after this is merged to be able to remove the field.